third_party/metrics_proto/PRESUBMIT.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2018 The Chromium Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+"""Presubmit script for metrics_proto.
+
+See http://dev.chromium.org/developers/how-tos/depottools/presubmit-scripts
+for more details on the presubmit API built into gcl.
+"""
+
+README = 'README.chromium'
```

Comment:
Not sure if this file is helpful either...?

Reply:
We aren't using those, as `PRESUBMIT.py` is a `depot_tools` construct, and we migrated away from DT. But there's also no reason to delete them, better to leave as is and modify upstream directory imports as little as possible.

Reply:
ack